internal/internal_task_handlers.go

```diff
@@ -633,6 +633,7 @@ func (wth *workflowTaskHandlerImpl) createWorkflowContext(task *workflowservice.
 		Namespace:                wth.namespace,
 		Attempt:                  attributes.GetAttempt(),
 		lastCompletionResult:     attributes.LastCompletionResult,
+		lastFailure:              convertFailureToError(attributes.ContinuedFailure, wth.dataConverter),
```

Comment:
Let's keep Failure at this level. DataConverter doesn't really belong here.

Reply:
Fixed